engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableAggregationTest.java

```diff
@@ -887,7 +887,7 @@ public void testKeyColumnTypes() {
                         new BigDecimalGenerator(),
                         new IntGenerator()));
 
-        final Set<String> keyColumnSet = new LinkedHashSet<>(table.getColumnSourceMap().keySet());
+        final Set<String> keyColumnSet = new LinkedHashSet<>(table.getDefinition().getColumnNames());
```

Comment:
I raised a discussion around a similar bit of code earlier. In that case, you chose to stream into a map collector. Here, you're using a newly-created list. Maybe both are better like this one, or using a `getColumnNameSet()` as the input.